themebuilder/modules/less-template-loader.js

```diff
@@ -65,7 +65,7 @@ class LessMetadataPostCompilerPlugin {
         if(this.swatchSelector) {
             const escapedSelector = this.swatchSelector.replace(".", "\\.");
 
-            const customStylesDuplicateRegex = new RegExp(`\\s+${escapedSelector}\\s+\.dx-theme-.*?-typography\\s+\.dx-theme-.*?{[\\s\\S]*?}[\\r\\n]*?`, "g");
```

Comment:
We need to use `\\.dx-theme`. `RegExp("\.")` generates `/./`, `RegExp("\\.")` - `/\./`

Reply:
@babich-a Thank you! Fixed.